docs/release.md

```diff
@@ -0,0 +1,180 @@
+# OSA release process
+
+OSA is designed to allow customers to choose an update `stream` for their cluster. Currently the only stream is `3.11`, but this may change over time. A single release of the OSA project can in principle support co-existent clusters at multiple streams. In principle, clusters will one day be upgradable between streams. This is not in scope for this document.
+
+An OSA release is a combination of multiple versioned items:
+
+Per stream:
+
+* VM image (operating system, openshift RPMs)
+* OpenShift container images
+* Microsoft container images (mdsd, mdm, etc.)
+* Openshift-azure container images (sync, etcdbackup, azure-controllers, etc.)
+* Openshift-azure repository, including plugin and manifest per stream
+
+For each stream, the configuration manifest combines the version numbers of all of the above (except the openshift-azure repository) and is checked into the openshift-azure repository. 
+It is set as `clusterVersion` field.
+
+OSA is an evolving service. During normal operations, we aim to release the RP infrastructure on a weekly basis, although emergency changes can be expedited.
+Globally, we aim for all production RP release versions to correspond.
+The OSA major release (`n`) is incremented with each weekly release.
+The OSA minor release (`n.x`) starts at .0 and is incremented by exception, e.g. to publish an emergency release (CVE, critical bug fix) or resolve a handover issue.
+
+OSA releases are defined by git tags on the openshift-azure repository (`vn.x`).
+The change to any production cluster implied by a new release varies. It may be nothing; alternatively it is also feasible that many or all versioned items could change in a release.
```

Comment:
consider: `The amount of changes per new release could vary.  This range includes little to no changes to many or possibly all versioned items changing.`